cmd/fiovb.c

```diff
@@ -0,0 +1,221 @@
+
+/*
+ * (C) Copyright 2018, Linaro Limited
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
+ */
+#include <common.h>
+#include <command.h>
+#include <env.h>
+#include <image.h>
+#include <malloc.h>
+#include <mmc.h>
+#include <fiovb.h>
+
+static struct FIOvbOps *fiovb_ops;
```

Comment:
Is camel case really supported by u-boot?

Reply:
nah I dont think so. looks horrible to me but that is what was merged so I thought it was better just to follow this as a guideline. anyway, will change it.